conf/variant/common/bblayers.conf

```diff
@@ -17,4 +17,9 @@ BBLAYERS ?= "                                          \
   ${BSPDIR}/sources/meta-virtualization                \
 "
 
+# Workaround to build b2qt and qt5.12 with rocko
```

Comment:
why do we need this in `master`?

Reply:
I also wonder this, since pelux master is synced with the yocto release thud. Are there parts of boot2qt for Qt 5.12 which are still on rocko, is that why?

Reply:
When building without I get
```
➜  sources bitbake core-image-pelux-qtauto-neptune-dev
ERROR: Unable to start bitbake server (None)
ERROR: Server log for this session (/home/yoctouser/pelux_changes/build/bitbake-cookerdaemon.log):
--- Starting bitbake server pid 5098 at 2019-03-26 14:44:07.877195 ---
Traceback (most recent call last):
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/cookerdata.py", line 284, in parseBaseConfiguration
    self.data = self.parseConfigurationFiles(self.prefiles, self.postfiles)
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/cookerdata.py", line 398, in parseConfigurationFiles
    % (c, " ".join(layerseries), " ".join(compat)))
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/__init__.py", line 120, in fatal
    raise BBHandledException()
bb.BBHandledException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/daemonize.py", line 83, in createDaemon
    function()
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/server/process.py", line 469, in _startServer
    self.cooker = bb.cooker.BBCooker(self.configuration, self.featureset)
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/cooker.py", line 210, in __init__
    self.initConfigurationData()
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/cooker.py", line 375, in initConfigurationData
    self.databuilder.parseBaseConfiguration()
  File "/home/yoctouser/pelux_changes/sources/poky/bitbake/lib/bb/cookerdata.py", line 317, in parseBaseConfiguration
    raise bb.BBHandledException
bb.BBHandledException
ERROR: Layer b2qt-distro is not compatible with the core layer which only supports these series: thud (layer is compatible with sumo)
```

Reply:
> why do we need this in `master`?

What branch do you suggest?

Reply:
what exactly are you building, PELUX? which version of the manifest do you use?